Formula/subversion.rb

```diff
@@ -30,10 +30,12 @@ class Subversion < Formula
   # build against Homebrew versions of
   # gettext, lz4, perl, sqlite and utf8proc for consistency
   depends_on "gettext"
+  depends_on :macos # Due to Python 2
+  # See https://github.com/Homebrew/homebrew-core/issues/53193#issue-600482673
+  # Will work with Python 3.8 in subversion 1.14 
```

Comment:
Is it just the Python bindings that need Python 2? Couldn't they just be installed on macOS only?

Reply:
Yes, I'll handle that case in linuxbrew when merging this, I'll revert back to Python 3 there.

I want to keep these lines (depends_on :macos # Due to Python 2) as it is easy to grep through the repo and find the Python 2 formulae.